lib/components/narrative/line-itin/place-row.js

```diff
@@ -156,42 +156,58 @@ class RentedVehicleLeg extends PureComponent {
   render () {
     const { config, leg } = this.props
     const configCompanies = config.companies || []
+
+    // Sometimes rented vehicles can be walked over things like stairs or other
+    // ways that forbid the main mode of travel.
     if (leg.mode === 'WALK') {
       return (
         <div className='place-subheader'>
           Walk vehicle along {leg.from.name}
         </div>
       )
     }
-    if (leg.rentedVehicle || leg.rentedBike || leg.rentedCar) {
-      let pickUpString = 'Pick up'
-      if (leg.rentedBike) {
-        // TODO: Special case for TriMet may need to be refactored.
-        pickUpString += ` shared bike`
-      } else {
-        // Add company and vehicle labels.
+
+    let pickUpString = 'Pick up'
```

Comment:
Since this variable sometimes does not contain `Pick up`, maybe we should rename it to `rentalString` or something.